### PR TITLE
Reunion 0.5.5 fixes

### PR DESCRIPTION
**Pull request by jeromelaban**
## PR Type

What kind of change does this PR introduce?
- Bugfix
- Build or CI related changes

## What is the new behavior?

- Update to latest 0.5.5 for generation.
- Fixes https://github.com/unoplatform/uno/pull/5786

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] Docs have been added/updated which fit [documentation template](https://github.com/unoplatform/uno/blob/master/doc/.feature-template.md) (for bug fixes / features)
- [ ] [Unit Tests and/or UI Tests](https://github.com/unoplatform/uno/blob/master/doc/articles/uno-development/working-with-the-samples-apps.md) for the changes have been added (for bug fixes / features) (if applicable)
- [ ] Validated PR `Screenshots Compare Test Run` results.
- [ ] Contains **NO** breaking changes
- [ ] Associated with an issue (GitHub or internal) and uses the [automatic close keywords](https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue).
- [ ] Commits must be following the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#summary) specification.

## Other information

Internal Issue (If applicable):

### CODE REVIEW

**Review by ajpinedam — APPROVED** [review]

File: src/SolutionTemplate/UnoSolutionTemplate/Shared/App.xaml.cs [hunk 1]
```diff
@@ -23,6 +23,8 @@ namespace $ext_safeprojectname$
     /// </summary>
     public sealed partial class App : Application
     {
+        private Window window;
```
Comment (davidjohnoliver) [inline src/SolutionTemplate/UnoSolutionTemplate/Shared/App.xaml.cs:26]:
```suggestion
        private Window _window;
```

**Review by davidjohnoliver — APPROVED** [review]

**Comment by jeromelaban** [comment]
Pausing merge because of https://github.com/microsoft/microsoft-ui-xaml/issues/4855

**Review by carldebilly — COMMENTED** [review]
I think the title of this PR should be `0.5.6` ;-)

**Comment by jeromelaban** [comment]
Superseeded by #5866

**Pull request closed without merging**